docs/infrastructure_and_maintenance/security/development_security.md

```diff
@@ -145,3 +145,18 @@ security:
                 entry_point: lexik_jwt_authentication.jwt_token_authenticator
             stateless: true
 ```
+
+In order for the application to be able to generate JWT tokens, a [PEM encoded keyset needs to be generated](https://symfony.com/bundles/LexikJWTAuthenticationBundle/current/index.html#generate-the-ssl-keys) using the command:
```

Comment:
```suggestion
Finish the setup by generating a [PEM encoded key pair](https://symfony.com/bundles/LexikJWTAuthenticationBundle/2.x/index.html#generate-the-ssl-keys) by using the command:
```
1) I'd use `key pair` consistently, imho no need to introduce "keyset" kere
2) Let's use the 2.x link for the 4.6 version (I will change it when upmerging)